src/YoutubeAtom.tsx

```diff
@@ -73,17 +73,6 @@ const constructQuery = (query: { [key: string]: any }): string =>
         })
         .join('&');
 
-type YoutubeMeta = {
-    assetId: string;
-    mediaTitle: string;
-    id?: string;
-    channelId?: string;
-    duration?: number;
-    posterSrc?: string;
-    height?: string;
-    width?: string;
```

Comment:
Does CAPI send any of this data or was it all redundant?

Reply:
its redundant, it was just not deconstructed into individual props